uploader/pantheon.py

```diff
@@ -250,6 +250,7 @@ def process_file(path, filetype):
         data = _generate_data(jcr_primary_type, base_name, path.name, asccidoc_type='nt:file')
         # This is needed to add a new module version, otherwise it won't be handled
         data[':operation'] = 'pant:newModuleVersion'
+        data['Content-Type'] = 'UTF-8'
```

Comment:
This approach is good for short term approach until we find how to pass charset as header with multipart data. However, use some other name rather than `content-type`. My suggestion is to use `encoding`

Reply:
renamed the parameter name to encoding